src/registrar/tests/test_views.py

```diff
@@ -1219,6 +1219,8 @@ def setUp(self):
         self.app.set_user(self.user.username)
         self.client.force_login(self.user)
 
+
+class TestDomainDetail(TestDomainOverview):
```

Comment:
**question (non-blocking)**: I saw these changes in another PR and they seem much less relevant here. To prevent merge conflicts, can we just take these changes out of this PR?

Reply:
if I take them out of the PR, unit tests fail in the pipeline.  i have to merge one of my branches into main and then I can remove these from each PR by merging main